.storybook/config.js

```diff
@@ -5,17 +5,26 @@ import '../packages/bpk-stylesheets';
 import TOKENS from './../packages/bpk-tokens/tokens/base.common';
 import BpkGridToggle from './../packages/bpk-component-grid-toggle';
 import BpkRtlToggle from './../packages/bpk-component-rtl-toggle';
+import BpkThemeSwitcher, { updateOnThemeChange } from './../packages/bpk-component-theme-switcher';
+import BpkThemeProvider from './../packages/bpk-theming';
+import themeAttributes from './themableAttributes';
 
-addDecorator(story => (
+const EnhancedThemeProvider = updateOnThemeChange(BpkThemeProvider);
+
+addDecorator((story, meta) => (
   <div style={{ padding: TOKENS.spacingBase }}>
-    {story()}
+    <EnhancedThemeProvider themeAttributes={themeAttributes[meta.kind]}>
+      {story()}
+    </EnhancedThemeProvider>
     <br />
     <BpkGridToggle />
     <br />
     <BpkRtlToggle />
+    <br />
+    <BpkThemeSwitcher />
```

Comment:
Why `ThemeSwitcher` as opposed to `ThemeToggle`?

Reply:
I thought the end plan was to have multiple themes or theme on the fly applied to storybooks and docs, so it'll switch to a new theme, if it make more sense to change it to toggle I'll do it

Reply:
I like toggle better :)

Reply:
Toggle it is!